### DISCUSSION TITLE
August 30 Quansight Show & Tell

### DISCUSSION THREAD

**Discussion by isabela-pf** [Quansight/Quansight.github.io#44] [Show and tell]
August 30 show & tell notes. Please use this discussion to mark highlights and links from our discussions.

Do not disclose confidential customer information.

The first attempt at show and tell generated by Tony's notebook.
technically the host: @isabela-pf 
Topic: packaging!

**Comment by isabela-pf** [comment]
Jaime: https://jaimergp.github.io/scientific-packaging-glossary/

Mahe: generating recipes! https://github.com/conda-incubator/grayskull#readme

Chris: conda store!! Links to come.

**Reply by saulshanabrook** [reply]
So cool! You can even generate one online: https://www.marcelotrevisani.com/grayskull

**Comment by isabela-pf** [comment]
Saul shared https://github.com/nix-community/nixpkgs-pytools/ in the chat!
Tony shared https://github.com/regro/cf-graph-countyfair in the chat!

**Reply by jaimergp** [reply]
And this one! https://github.com/Quansight-Labs/beni

**Comment by isabela-pf** [comment]
Today I learned: https://en.wikipedia.org/wiki/Slowloris_(computer_security) and that this is also the name of an animal.

**Reply by jaimergp** [reply]
The ASCII art is ❤️ 

![image](https://user-images.githubusercontent.com/2559438/131375608-7cb4e677-3ee9-415d-a06b-b3a37ca83cc7.png)

**Reply by isabela-pf** [reply]
🥺 energy
